common.go

```diff
@@ -115,8 +113,12 @@ func FormatErrors(errs []OpsLevelErrors) error {
 	return fmt.Errorf(strings.Join(errstrings, "\n"))
 }
 
+// NewId use "" to set "null" for ID input fields that can be nullified
 func NewID(id string) *graphql.ID {
-	output := graphql.ID(id)
+	var output graphql.ID
```

Comment:
I had to make this change so I could have update mutations set ID's to a json value of "null" so you can properly "unset" something in an Update mutation